### PR TITLE
Update Google Cast docs

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Update Google Cast docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by bdraco — DISMISSED** [review]

**Comment by dgwynne** [comment]
FWIW, this looks good to me.

File: source/_integrations/cast.markdown [hunk 1]
```diff
@@ -182,11 +182,17 @@ data:
         - url: "https://tilos.hu/images/kockalogo.png"
 ```
 
-## Cast devices and Home Assistant on different subnets
+## Trouble shooting automatic discovery
+mDNS relies on UDP multicast, which may fail for various reasons. If none of the tips in this section helps, the recommended solution is to ensure all cast devices have static IPs assigned to them and configure a list of known hosts.
```
Comment (frenck) [inline source/_integrations/cast.markdown]:
```suggestion
## Troubleshooting automatic discovery

mDNS relies on UDP multicast, which may fail for various reasons. If none of the tips in this section helps, the recommended solution is to ensure all cast devices have static IPs assigned to them and configure a list of known hosts.
```

File: source/_integrations/cast.markdown [hunk 2]
```diff
@@ -182,11 +182,17 @@ data:
...
 
-Cast devices can only be automatically discovered if they are on the same subnet as Home Assistant.
-Setups with cast devices on a different subnet than Home Assistant are not recommended and not supported.
-If this is not possible, it's necessary to either enable mDNS forwarding between the subnets or to provide a list of known hosts.
+### Zeroconf configuration
+The Google Cast integration relies on the [Zeroconf integration](/integrations/zeroconf) for mDNS discovery. The Zeroconf integration has some configuration options which impact mDNS routing.
```
Comment (frenck) [inline source/_integrations/cast.markdown]:
```suggestion
### Zeroconf configuration

The Google Cast integration relies on the [Zeroconf integration](/integrations/zeroconf) for mDNS discovery. The Zeroconf integration has some configuration options which impact mDNS routing.
```

File: source/_integrations/cast.markdown [hunk 3]
```diff
@@ -182,11 +182,17 @@ data:
...
+
+### Cast devices and Home Assistant on different subnets
+
+Cast devices can only be automatically discovered if they are on the same subnet as Home Assistant because mDNS packets are not routed across subnets.
+Automatic discovery in setups with cast devices on a different subnet than Home Assistant are not recommended and not supported.
```
Comment (frenck) [inline source/_integrations/cast.markdown]:
```suggestion
Automatic discovery in setups with cast devices on a different subnet than Home Assistant is not recommended and not supported.
```

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery 👍

**Pull request merged**